src/extensions/default/InlineTimingFunctionEditor/TimingFunctionUtils.js

```diff
@@ -42,7 +42,7 @@ define(function (require, exports, module) {
         BEZIER_CURVE_GENERAL_REGEX      = /cubic-bezier\((.*)\)/,
         EASE_STRICT_REGEX               = /[: ,]ease(?:-in)?(?:-out)?[ ,;]/,
         EASE_LAX_REGEX                  = /ease(?:-in)?(?:-out)?/,
-        LINEAR_STRICT_REGEX             = /transition.*?[: ,]linear[ ,;]/,
+        LINEAR_STRICT_REGEX             = /transition|animation.*?[: ,]linear[ ,;]/,
```

Comment:
This should actually be between parenthesis, like this: `(transition|animation).*?[: ,]linear[ ,;]`